magento/magento2
# [MFTF] Refactoring of AdminConfigDefaultProductLayoutFromConfigurationSettingTest

### Description (*)
The test is refactored according to the best practices followed by MFTF.

### Manual testing scenarios (*)
1. Go to Admin->Stores->Configuration->General->Web->Default layouts
2. Set Default Product Layout to "3 column" and sve changes
3. Start creating a new simple product
4. Check if Design->Layout is "3 column"




### Resolved issues:
1. [x] resolves magento/magento2#30855: [MFTF] Refactoring of AdminConfigDefaultProductLayoutFromConfigurationSettingTest
